proxmox/access/realm.go

```diff
@@ -0,0 +1,91 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package access
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"sort"
+
+	"github.com/bpg/terraform-provider-proxmox/proxmox/api"
+)
+
+func (c *Client) RealmsPath() string {
+	return c.ExpandPath("domains/")
```

Comment:
This is the reason for 403 error response in `api_test.go`
```suggestion
	return c.ExpandPath("domains")
```

Reply:
I'm fairly certain I tried it with&without the tailing `/`. I'll give it another go, maybe I wasn't paying attention